absl/debugging/internal/stacktrace_aarch64-inl.inc

```diff
@@ -37,9 +37,11 @@ static const unsigned char* GetKernelRtSigreturnAddress() {
   absl::debugging_internal::VDSOSupport vdso;
   if (vdso.IsPresent()) {
     absl::debugging_internal::VDSOSupport::SymbolInfo symbol_info;
-    if (!vdso.LookupSymbol("__kernel_rt_sigreturn", "LINUX_2.6.39", STT_FUNC,
-                           &symbol_info) ||
-        symbol_info.address == nullptr) {
+    auto lookup = [&](int type) {
+      return vdso.LookupSymbol("__kernel_rt_sigreturn", "LINUX_2.6.39", type,
+                               &symbol_info);
+    };
+    if ((!lookup(STT_NOTYPE) && !lookup(STT_FUNC)) || symbol_info.address == nullptr) {
```

Comment:
Can you invert the order of `STT_FUNC` and `STT_NOTYPE`?

I expect `STT_FUNC` to be more common (and correct), so we should look for it first.
Otherwise looks good to me.

Reply:
Thanks! Fixed.